src/Native/LdaNative/data_block.cpp

```diff
@@ -45,8 +49,12 @@ namespace lda
 
     void LDADataBlock::Allocate(const int32_t num_document, const int64_t corpus_size)
```

Comment:
If we change `cropus_size` to `size_t`, do we still need the following check?

Reply:
Yes, I can not change the corpus size to size_t because that value changes per platform bitness (32/64)  I do not have a mechanism to support this in C# / C++ interface.

---
In reply to: [276746121](https://github.com/dotnet/machinelearning/pull/3370#discussion_r276746121) [](ancestors = 276746121)